lib/cli/file-set-pipeline/configure.js

```diff
@@ -28,16 +28,16 @@ var Configuration = require('../configuration');
  * @param {CLI} cli
  */
 function configure(cli) {
-    var configuration = new Configuration({
-        'detectRC': cli.detectRC,
-        'file': cli.configPath,
-        'settings': cli.settings,
-        'plugins': cli.plugins,
-        'output': cli.output,
-        'cwd': cli.cwd
-    });
-
-    cli.configuration = configuration;
+    if (!cli.configuration) {
```

Comment:
Ah OK, thanks for the clarification. Now it makes sense.

Yeah, we should keep these changes.